build-tools/packages/build-tools/src/repoPolicyCheck/handlers/index.ts

```diff
@@ -0,0 +1,18 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { assertShortCodeHandler, copyrightFileHeaderHandlers, dockerfilePackageHandler, fluidCaseHandler, Handler, lockfilesHandlers, npmPackageContentsHandlers } from "../..";
```

Comment:
import these individually from the source files. e.g.

```ts
import {assertShortCodeHandler} from "./assertShortCode";
```

What you're doing here is creating a new module, and the API for the module is defined by what's exported from this file (index.ts). The individual handlers are "private" to the module, so you import them here and then export them in the shape that you want for your "external/public" API for the module. When you make this change and remove the exports from the root index.ts, you'll effectively "hide" all the individual handlers from the outside world. Even code within the project but outside the module won't be able to access them directly without a lint error. All access will be through the array exported here.

Reply:
Sorry, I was in the process of doing that